### ISSUE TITLE
InfluxDB: parallel banks are receiving first bank's voltages

### ISSUE THREAD

**Issue by jeronimo** [stuartpittaway/diyBMSv4Code#123]
<img width="485" alt="Screenshot 2021-10-18 at 15 45 09" src="https://user-images.githubusercontent.com/88912/137804228-a3b53ab2-790e-497f-b0e3-bab509dfdeda.png">
<img width="1014" alt="Screenshot 2021-10-18 at 15 45 17" src="https://user-images.githubusercontent.com/88912/137804229-d4dcff39-4a50-422c-af90-ad010f88108a.png">

`diyBMSv4Code Release-2021-03-29-15-35`
`InfluxDB shell version: 1.6.4`

**Comment by jeronimo** [comment]
Just wrote a quick script to send data from Victron Smartshunt via Ruby to InfluxDB and have started to look at diybms code as well. It seems that in https://github.com/stuartpittaway/diyBMSv4Code/blob/87f6de125b10ed8b0e9e4830e15a79ecb33127d8/ESPController/src/main.cpp#L725 array indexes should be `i+bank` though not sure if `cmi` array already has all the values.
If it is not a quick fix I will make a PR at some point :)

**Comment by stuartpittaway** [comment]
Hello @jeronimo just checking that you are reporting this issue against the ESP8266 version of the DIYBMS controller?

This issue was resolved in the ESP32 version.

**Comment by jeronimo** [comment]
Yes, I am still using old controller with Wemos D1 Mini ESP8266.

**Comment by rolfbartels** [comment]
Just confirming that this issue still exists

**Comment by stuartpittaway** [comment]
Although @rolfbartels is using the ESP32 version which has a different code base

**Comment by rolfbartels** [comment]
Sorry, just to calrify I am using the old controller with the Wemos D1 Mini ESP8266

**Comment by stuartpittaway** [comment]
Ah, in which case that's a whole different ball game!

**Comment by stuartpittaway** [comment]
@rolfbartels just updated the code, do you want to try that one out?

**Comment by rolfbartels** [comment]
@stuartpittaway I can confirm, this has fixed the error, thank you
